docs/guide/connecting.rst

```diff
@@ -7,30 +7,97 @@
 Connecting to Verilated Models
 ******************************
 
+Structure of the Verilated Model
+================================
+
+Verilator outputs a :file:`{prefix}.h` header file which defines a class
+named :code:`{prefix}` which represents the generated model the user is
+supposed to instantiate.  This Model Class defines the interface of the
+Verilated model.
+
+Verilator will additionally create a :file:`{prefix}.cpp` file, together
+with additional .h and .cpp files for internals.  See the :file:`examples`
+directory in the kit for examples.  See :ref:`Files Read/Written` for
+information on all the files Verilator might output.
+
+The output of Verilator will contain a :file:`{prefix}.mk` file that may be
+used with Make to build a :file:`{prefix}__ALL.a` library with all required
+objects in it.
+
+The generated Model Class file manages all internal state required by the
```

Comment:
```suggestion
The generated model class file (:file:`{prefix}.h`) manages all internal state required by the
```

Reply:
I have been using Model Class as a proper noun in caps everywhere, so do I change all or none?

Reply:
Lower case consistently please.  Agree it's somewhat arbitrary, but it's the convention I request with the technical writers on my real-world-job team, so what I'm used to.

Reply:
No probs, indeed it's just convention.